### PR TITLE
changed timeout to 24h temporarily

### PR DESCRIPTION
**Pull request by yliaog**
changed timeout to 24h to see if tests show up on testgrid

### CODE REVIEW

**Comment by yliaog** [comment]
https://github.com/kubernetes/test-infra/issues/15267

**Comment by BenTheElder** [comment]
You all should set up OWNERS for these job configs 🙃

File: config/jobs/kubernetes/sig-windows/windows-gce.yaml [hunk 1]
```diff
@@ -166,7 +166,7 @@ periodics:
       - --cluster=
       - --extract=ci/k8s-master
       - --gcp-zone=us-west1-b
-      - --ginkgo-parallel=1
+      - --ginkgo-parallel=8
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-windows/windows-gce.yaml]: I don't see anything excluding serial tests that can't run in parallel, I forget, will kubetest do that for you?

**Comment by yliaog** [comment]
could you please approve?

**Pull request merged**